README.md

```diff
@@ -1,9 +1,71 @@
 # birdweather_daily_email
 
+![example email](/img/birds-email-1.png "Example email sent by the program")
+
 A personal project for sending a daily summary of the birds whose calls were identified by my stations on [BirdWeather](https://app.birdweather.com). And also learning some basic Go.
```

Comment:
i might add a little more detail here.  The rest of the doc is how to set it up and use it, but even a personal touch about your kids looking at the email each day or something.  I *love* the last sentence, since that's the real reason!